src/elements/db/SubscriptionQuery.php

```diff
@@ -588,19 +588,21 @@ public function onTrial(?bool $value = true): SubscriptionQuery
     /**
      * Narrows the query results based on the subscriptions’ next payment dates.
      *
+     * Values must be a unix timestamp.
+     *
      * Possible values include:
      *
      * | Value | Fetches {elements}…
      * | - | -
-     * | `'>= 2018-04-01'` | with a next payment on or after 2018-04-01.
-     * | `'< 2018-05-01'` | with a next payment before 2018-05-01
-     * | `['and', '>= 2018-04-04', '< 2018-05-01']` | with a next payment between 2018-04-01 and 2018-05-01.
+     * | `'>= 1522558800'` | with a next payment on or after 1522558800 (2018-04-01).
+     * | `'< 1525150800'` | with a next payment before 1525150800 (2018-05-01)
+     * | `['and', '>= 1522558800', '< 1525150800']` | with a next payment between 1522558800 (2018-04-01) and 1525150800 (2018-05-01).
      *
      * ---
      *
      * ```twig
      * {# Fetch {elements} with a payment due soon #}
-     * {% set aWeekFromNow = date('+7 days')|atom %}
+     * {% set aWeekFromNow = date('+7 days')|format('u') %}
```

Comment:
I think that’s supposed to be`date('+7 days')|date('U')`?

Reply:
Yep, this is what happens when I mix up my Twig and PHP